sdks/java/extensions/sql/src/test/thrift/kafka/message.thrift

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+thrift --gen java:private-members \
```

Comment:
+1 for this comment - it'll be helpful :)

Reply:
I initially tried to generate java code from thrift "on the fly" using the `thrift-gradle-plugin`, but it turns out `thrift` is not installed on the Jenkins servers so the build fails. That's why I had to document how the code is to be generated manually.

Reply:
Yeah, I read the rest of the comments, that's why it's neat to have this comment here :)